### DISCUSSION TITLE
Nodes showing out of sync

### DISCUSSION THREAD

**Discussion by otterslide** [Chia-Network/chia-blockchain#13975] [Support]
I usually see all nodes I'm connected to in sync with the latest block, but recently I'm seeing 50% of nodes out of sync.
Is this just a GUI issue, or are those nodes really out of sync? And if they are not, why isn't my node updating them?
This has been going on for hours now, .. 50% of nodes are keeping up and the rest are behind or way behind. They update once in a while, but stay behind.
![image](https://user-images.githubusercontent.com/11649943/203130960-1e4bd82c-75f1-46e8-83a8-c4ce1ec049d6.png)

**Comment by wallentx** [comment]
If you are familiar with using the CLI? It would be interesting to see if the nodes that are showing out of date in the GUI also show up as connected clients when you run `chia peer full_node -c`. It may be that clients are disconnecting, but not getting removed from the GUI.
Of the ones that are behind, do you know if the IPs/Node IDs are the same?
Is your node having any issues staying synced?

**Reply by otterslide** [reply]
> If you are familiar with using the CLI? It would be interesting to see if the nodes that are showing out of date in the GUI also show up as connected clients when you run `chia peer full_node -c`. It may be that clients are disconnecting, but not getting removed from the GUI. Of the ones that are behind, do you know if the IPs/Node IDs are the same? Is your node having any issues staying synced?

I will try that next time. As of right now, they're all synced again, and all those IPs have disconnected. I checked with xchscan's latest block and it seemed like I was on the latest block. It's hard to tell since it takes about 30 sec for the website to update. Never lost sync according to Chiadog for days. Is the client smart enough to detect lost sync with the latest block if all the nodes it's connected to are behind?
I believe it's possible to be out of sync, without being notified of being out of sync. I was thinking some kind of redundancy for sync checking could be added, such as an online webservice the client can call to get the latest block. Someone would have to run that server,.. but I think it could help a lot of farmers that are connecting to very few nodes and falling behind without knowing it.

![image](https://user-images.githubusercontent.com/11649943/203365970-d8aabad4-dfb7-4607-991b-a36ed49bb767.png)
